openlibrary/components/MergeUI/AuthorRoleTable.vue

```diff
@@ -15,9 +15,10 @@
                             {{(role[field].key || role[field]).slice("/type/".length)}}
                         </div>
                         <div v-else-if="field == 'author'">
-                            <a :href="`${role[field].key}`" target="_blank">
+                            <a :href="`${role[field].key}`" target="_blank" class="author-author-children">
                                 {{role[field].key.slice("/authors/".length)}}
                             </a>
+                            <p class="author-author-children"><b>{{role[field].name}}</b></p>
```

Comment:
Let's put this in a new `<td>` element so it appears as its own column on the nested author roles table.

Reply:
Oh actually, if we attach the author name to `entry` (eg the thing that has the `author` field) instead of to the `entry.author`, it should automatically show up! That might be easier.

Reply:
Eg

`entry.name = author_names[entry.author.key];`

instead of 

`entry.author.name = author_names[entry.author.key];`

Reply:
I'm pretty sure I understand why this should work (should be captured under the ```else``` statement) but I couldn't get the name to render without making an ```else-if``` branch for it.

Reply:
(Responded here: https://github.com/internetarchive/openlibrary/pull/9920#discussion_r1877234312 )